app/services/editions/db/CollectionsQueries.scala

```diff
@@ -191,4 +193,39 @@ trait CollectionsQueries {
 
   private case class GetCollectionsRow(collection: EditionsCollection, card: Option[DbEditionsCard])
 
+  /**
+    * Insert a collection owned by the specified front.
+    *
+    * @return the Collection id
+    */
+  protected def insertCollection(frontId: String, collectionIndex: Int, name: String, now: OffsetDateTime, user: User)(implicit session: DBSession) =
+    Try {
+      sql"""
+        INSERT INTO collections (
+          front_id,
+          index,
+          name,
+          is_hidden,
+          updated_on,
+          updated_by,
+          updated_email
+        ) VALUES (
+          $frontId
+          , ${collectionIndex}
+          , $name
```

Comment:
Oooh - looks like this might be vulnerable to SQL injection?  For example, what would happen if I called the endpoint with `?name=test, FALSE, \"2024-01-02T03:04:05Z\", \"evil-hacker\", \"evil-hacker@pwned.net\"); drop table collections;` ? (appropriately URL-encoded, of course!)

Unless the `sql` modifier on the string is doing proper placeholder conversion of course?

Reply:
_> Unless the sql modifier on the string is doing proper placeholder conversion of course?

👍 😀   [(docs)](https://scalikejdbc.org/documentation/sql-interpolation.html)

Reply:
😌